chess-AI/stockfish_train.py

```diff
@@ -15,31 +19,22 @@ class StockfishTrain:
         stockfish: the stockfish object
     """
 
-    def __init__(self, path):
-        self.stockfish = Stockfish(path)
+    def __init__(self):
+        self.stockfish = util.create_stockfish_wrapper()
 
     def set_params(self):
         """Sets the elo and depth for stockfish using the dashboard
 
         Attributes:
           dashboard: reference to the streamlit dashboard (this is temporary)
         """
-        
+
         # Set the elo and depth of the stockfish object
         self.stockfish.set_elo_rating(1000)
         self.stockfish.set_depth(3)
```

Comment:
idk if this should go in a separate pr but elo and depth should be constructor arguments imo, i dont think set_params should exist

Reply:
Agree, I'll do it in this one